src/quantum/eng/Generate-Vendored-SDKs.ps1

```diff
@@ -0,0 +1,116 @@
+The following clients can be generated bby this script:
+- Azure Quantum Python data-plane client
+- Azure Quantum Python control-plane client
+- Azure Storage Blob client
+.DESCRIPTION
+Generate the "vendored-sdks" clients for the CLI based on the latest published Swagger.
+The following clients can be generated bby this script:
+- Azure Quantum Python data-plane client
+- Azure Quantum Python control-plane client
+- Azure Storage Blob client
+.PARAMETER SwaggerRepoUrl
+The URL of the git repo that contains the Swagger and AutoRest ReadMe.md configurations (defaults to "https://github.com/Azure/azure-rest-api-specs")
+.PARAMETER SwaggerRepoBranch
+The name of the swagger repo branch (defaults to "main")
+.PARAMETER SwaggerTagVersion
+The Swagger version to be used (defaults to "", which will use the default tag from the main ReadMe.md)
+.PARAMETER ClientToGenerate
+Select which client to generate: quantum data-plane, quantum control-plane, Azure storage blob (defaults to "quantum data-plane")
+
+.EXAMPLE
+./eng/Generate-Client.ps1
+
+# Regenerate the data-plane client using the latest published Swagger from the official repo
+
+.EXAMPLE
+./eng/Generate-Client.ps1 -SwaggerRepoBranch "feature/quantum/update-clients"
+
+# Regenerate the data-plane client using the Swagger from the official repo, but from a feature branch
+
+.EXAMPLE
+./eng/Generate-Client.ps1 -SwaggerTagVersion "package-2019-11-04-preview"
+
+# Regenerate the data-plane client using the an older version of the Swagger
+
+.EXAMPLE
+./eng/Generate-Client.ps1 -ClientToGenerate "control-plane"
+
+# Regenerate the control-plane client using the latest published Swagger from the official repo
+
+#>
+
+[CmdletBinding()]
+Param (
+    [string] $SwaggerRepoUrl = "https://github.com/Azure/azure-rest-api-specs",
+    [string] $SwaggerRepoBranch = "main",
+    [string] $SwaggerTagVersion,
+    [string] $ClientToGenerate = "data-plane"
+)
+
+# Select which client to generate
+if ([string]::IsNullOrEmpty($ClientToGenerate) -or ($ClientToGenerate -eq "data-plane"))
+{
+    $AutoRestConfig = "$SwaggerRepoUrl/blob/$SwaggerRepoBranch/specification/quantum/data-plane/readme.md"
+    $OutputFolder = "./azext_quantum/vendored_sdks/azure_quantum/"
+}
+else
+{
+    if ($ClientToGenerate -eq "control-plane")
+    {
+        $AutoRestConfig = "$SwaggerRepoUrl/blob/$SwaggerRepoBranch/specification/quantum/resource-manager/readme.md"
+        $OutputFolder = "./azext_quantum/vendored_sdks/azure_mgmt_quantum/"
+    }
+    else
+    {
+        if ($ClientToGenerate -eq "storage-blob")
+        {
+            $AutoRestConfig = "$SwaggerRepoUrl/blob/$SwaggerRepoBranch/specification/storage/data-plane/Microsoft.BlobStorage/readme.md"
+            $OutputFolder = "./azext_quantum/vendored_sdks/azure_storage_blob/"
+        }
+        else
+        {
+            Write-Error "ERROR: ClientToGenerate parameter not recognized."
+            return -1
+        }
```

Comment:
Do we still generate the blob client this way? From the wiki I understood that we're copying it over from PyPi package as is. If this is not currently used, might be we could remove this code to not confuse ourselves later on how to get vendored_sdk for `azure_storage_blob`

Reply:
No, we will not generate the client files this way.  I intended to delete this script.  Deleting it now.